internal/server/server.go

```diff
@@ -209,6 +213,25 @@ func (ws *HTTPServer) Start() {
 	}
 }
 
+func (ws *HTTPServer) api(pattern string) string {
+	contextPath := ws.config.ServiceConfig.HTTPServer.ContextPath
+	if !strings.HasSuffix(contextPath, "/") {
```

Comment:
In s3 context I had the same problem and used oneliner
`strings.TrimSuffix(prefix, "/") + "/")`
wdyt?